site/Dance/index.html

```diff
@@ -0,0 +1,73 @@
+  </div>
+  <div>
+    <div>Curriculum</div>
+    <div>Shows</div>
+    <div>Faculty</div>
+    <div>Alumni</div>
+
+  </div>
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit
+    tristique quam. Donec mollis auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu dui. Duis
+    dictum in ligula a aliquet. Vestibulum consequat nisl a elementum tincidunt. Nunc nec egestas erat. Fusce efficitur,
+    nibh nec convallis eleifend, est lorem tristique quam, sit amet aliquet elit lorem eu quam. Etiam pharetra aliquam
+    orci, vitae blandit nisi maximus in. Integer ante eros, elementum vitae hendrerit eget, pharetra eget quam. Donec
+    fringilla, nisi nec fermentum pellentesque, ante eros consectetur ante, at venenatis ex dui ut velit.
+
+  </p>
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit
+    tristique quam. Donec mollis auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu dui. Duis
+    dictum in ligula a aliquet. Vestibulum consequat nisl a elementum tincidunt. Nunc nec egestas erat. Fusce efficitur,
+    nibh nec convallis eleifend, est lorem tristique quam, sit amet aliquet elit lorem eu quam. Etiam pharetra aliquam
+    orci, vitae blandit nisi maximus in. Integer ante eros, elementum vitae hendrerit eget, pharetra eget quam. Donec
+    fringilla, nisi nec fermentum pellentesque, ante eros consectetur ante, at venenatis ex dui ut velit.
+    <p>TBD</p>
+  </p>
+  <h2>Related Pages</h2>
+  <div>
+    <div>April 15</div>
+    <div>Dance Audtions</div>
+    <p>consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit tristique quam. Donec mollis
+      auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu dui. </p>
+    <a>Learn More</a>
+    <div>Alum Jenny Lim Joins Alvin Alley</div>
+    <p>consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit tristique quam. Donec mollis
+      auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu dui. </p>
+    <a>Learn More</a>
+    <div>May 23</div>
+    <div>BAA Students Perform the Nutcracker</div>
+    <p>consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit tristique quam. Donec mollis
+      auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu dui. </p>
+    <a>Learn More</a>
+    <div>George Hellman, '24, Wins Dance Competition</div>
+    <p>consectetur adipiscing elit. Aliquam augue dolor, luctus eu massa non, blandit tristique quam. Donec mollis
+      auctor vulputate. Aenean eu nisi sit amet diam consequat pulvinar interdum eu dui. </p>
+    <a>Learn More</a>
+  </div>
+  <div>
+    <footer>Footer TBD</footer>
+    <div>Newsletter Signup?</div>
+    <div>Social Media Icons</div>
+    <div>Connect</div>
+    <div>Apply</div>
+    <div>Give</div>
+    <div>Shop</div>
+  </div>
```

Comment:
I think you want the `<footer>` tag wrapping all this content. Right now the `<footer>` tag is just around "Footer TBD". Instead what I think you want is:

```
<footer>
    <div>Footer TBD</div>
    <div>Newsletter Signup?</div>
    <div>Social Media Icons</div>
    <div>Connect</div>
    <div>Apply</div>
    <div>Give</div>
    <div>Shop</div>
</footer>
```